### ISSUE TITLE
Feature Request: 1/3 and 2/3 size

### ISSUE THREAD

**Issue by amb-it-ion** [hoppfrosch/WindowPadX#15]
Similar to WinSplitRevolution, it would be nice to have multiple window pad strokes (e.g. win+Num4 first default behaviour, then 1/3 to the left, then 2/3 to the left).

**Comment by yorkshirelandscape** [comment]
Agreed. It would be nice if Win+Num5 would also cycle through multiple sizes before/instead of switching windows.

**Comment by alapo** [comment]
Perhaps it would be possible to implements [parts of this script](http://git.the-compiler.org/wingravity/tree/wingravity.ahk) into WindowPad

**Comment by alapo** [comment]
I figured it out. Open up `WindowPad.ini` with Notepad++ and add the following lines of code
`<#!Numpad4 = WPM, -1,  0,  0.66, 1.0
<#!Numpad6 = WPM, +1,  0,  0.66, 1.0
`

This might not be exactly what you are looking for but its a step in the right direction
